tests/e2e/editor-selector.spec.js

```diff
@@ -0,0 +1,38 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('Test the creation of a new Post', async ({page, context}) => {
```

Comment:
I see actually that in the initial test the [description](https://github.com/greenpeace/planet4-master-theme/blob/main/tests/acceptance/editor.feature#LL1C1-L1C1) is `editor basics` so maybe we could rename this one to `test editor basic functionalities` or something like this?